pages/ecosystem.vue

```diff
@@ -297,9 +298,11 @@ const filteredMembers = computed<Member[]>(() => {
 const filteredMembersSorted = computed<Member[]>(() => {
   if (selectedSortingOption.value === "stars") {
     return filteredMembers.value.sort((a, b) => b.stars - a.stars);
+  } else if (selectedSortingOption.value === "name") {
+    return filteredMembers.value.sort((a, b) => a.name.localeCompare(b.name));
   }
 
-  return filteredMembers.value.sort((a, b) => a.name.localeCompare(b.name));
+  return filteredMembers.value.sort((a, b) => b.tier.localeCompare(a.tier));
```

Comment:
I think that since the inverse alphabetical order (M, E, C) **only coincidentally** matches our "default" order logic we should do one of the following:

- either replace the alphabetical order with an explicit order matching "Main", "Ecosystem" and "Community", or
- add a comment similar to `// HACK: This sorting matches the default order: "Main", "Ecosystem", "Community"` for better code clarity and maintainability

For all intends and purposes of this PR, the second option seems enough to me.

Reply:
updated [here](https://github.com/Qiskit/qiskit.org/pull/3478/commits/b64976347e6f182f05b96a45b4f65ead7d93cee2)